### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Dmytro-Kretov**
[DEMO LINK](https://dmytro-kretov.github.io/react_static-list-of-posts-js/)

### CODE REVIEW

File: src/components/CommentInfo/CommentInfo.jsx [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{comment.name}</strong>
+
+        {' by '}
+
+        <a
+          className="CommentInfo__email"
+          href={mail}
+        >
+          {comment.email}
```
Comment (volodymyr-soltys97) [inline src/components/CommentInfo/CommentInfo.jsx:15]: Use a destruction for `comment`

File: src/components/CommentList/CommentList.jsx [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  comments.length
+    ? (
+      <div className="CommentList">
+        {comments.map(comment => (
+          <CommentInfo comment={comment} />
```
Comment (volodymyr-soltys97) [inline src/components/CommentList/CommentList.jsx:9]: Where is the key attribute here?

File: src/components/PostInfo/PostInfo.jsx [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
```
Comment (volodymyr-soltys97) [inline src/components/PostInfo/PostInfo.jsx:9]: Use a destruction for `post`

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

**Review by lerastarynets — APPROVED** [review]
You're showing incredible skills, keep it going!

**Pull request closed without merging**